.vib/jaeger/cypress/cypress/integration/jaeger_spec.js

```diff
@@ -0,0 +1,31 @@
+/// <reference types="cypress" />
+
+it('list and retrieve jaeger traces', () => {
+
+    const testService = 'redis';
+    const currentDate = new Date(); 
+    const timestampMillis = currentDate. getTime() * 1000;
+    // Check application availability though the UI
+    cy.visit('/search?end=' + timestampMillis + '&limit=20&lookback=1h&maxDuration&minDuration&service=' + testService + '&start=0').then(() => {
+
+        // Ensure page contains Traces in an H2 tag
+        cy.contains('h2', 'Traces');
```

Comment:
This line actually checks if there are Traces in the application (if there are traces the UI shows and `h2` with the text `Traces`). Is truth that I got a trace just below so the test would fail anyway so I will delete it.

Reply:
[c1724da](https://github.com/bitnami/charts/pull/13480/commits/c1724da122a105cae2ffd83a733f6723268200b6)